umbraco/Umbraco-CMS
# Content API: Get by controllers

## Details
- Creates the baseline for the Content API "get by" controllers: `/content/{id:guid}` and `/content/{url}`.
  - GET `/content/{id:guid}` - gets a content item by id;
  - GET `/content/{url}` - gets a content item by route;
    - For now, collisions are not considered - only the "happy" scenarios are accounted for.

## Test
- Set up some content in the backoffice:
  - A couple of root nodes and a few nested ones;
- Using Swagger (_the new endpoints can be found under **Content** for now_) or Postman:
  - GET `/content/{id:guid}`
    - Get an `id` value of a content node from the backoffice;
    - Pass it to GET `localhost/umbraco/content/api/v1/content/{id:guid}`;
    - Make sure the information in the response is correct for the content item with that `id`.
  - GET `/content/{url}`
    - Pass the `url` value for a content item to GET `localhost/umbraco/content/api/v1/content/{url}`;
    - Make sure the information in the response is correct for the content item with that `url`.
